docker/regtest/dockerfile-deps/joinmarket/latest/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM python:3.9.7-slim-bullseye
+
+RUN apt-get update && \
+    apt-get install -qq --no-install-recommends curl tini sudo procps vim git iproute2 \
+    build-essential automake pkg-config libtool libgmp-dev libltdl-dev python3-dev virtualenv python3-pip supervisor && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV REPO https://github.com/JoinMarket-Org/joinmarket-clientserver
+ENV REPO_REF master
+
+WORKDIR /src
+RUN git clone "$REPO" . --depth=10 --branch "$REPO_REF" && git checkout "$REPO_REF"
+
+RUN ./install.sh --disable-secp-check --without-qt
+ENV DATADIR /root/.joinmarket
+ENV CONFIG ${DATADIR}/joinmarket.cfg
+ENV DEFAULT_CONFIG /root/default.cfg
+ENV DEFAULT_AUTO_START /root/autostart
+ENV AUTO_START ${DATADIR}/autostart
+ENV ENV_FILE "${DATADIR}/.env"
+
+COPY default.cfg "${CONFIG}"
+
+RUN . jmvenv/bin/activate && cd /src/scripts && \
+    pip install matplotlib && \
+    (python wallet-tool.py generate || true) \
+    && cp "${CONFIG}" "${DEFAULT_CONFIG}"
```

Comment:
This is part of btcpayservers approach to substitute config variables. The `docker-entrypoint.sh` script will use the default config if necessary. Since we need to activate some config properties that are commented out the original config, which can not be substitute otherwise, we'll first need to bring our own config in, which is one of the few deviations from the original approach. I wanted it to be as close as possible with minimum changes.